zeppelin-interpreter-parent/pom.xml

```diff
@@ -113,6 +113,7 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-shade-plugin</artifactId>
           <configuration>
+            <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
It's a bit complicated but let me explain it.

When we install jar, `maven-instll-plugin` tries to install the file under `/target/*.jar` into `.m2` but our shade script store shaded jars under /interpreter`/{interpreter.name}` so the file under /target directory is an original file which is not shaded. so maven-install-plugin copy the original file into .m2 directly as a result. BTW, in case of pom file, dependency-reduced-pom should be copied into .m2 directory even if the jar is not shaded. In our old version of shade plugin, the default setting didn't create dependency-reduced-pom when using `outputFile` configuration but the latest one, the plugin creates even if we use `outputFile` setting. So I disabled to create it.